test/e2e/framework.go

```diff
@@ -643,22 +644,42 @@ func (data *TestData) deleteTestNamespace(timeout time.Duration) error {
 }
 
 // deployAntreaCommon deploys Antrea using kubectl on the control-plane Node.
-func (data *TestData) deployAntreaCommon(yamlFile string, extraOptions string, waitForAgentRollout bool) error {
+func (data *TestData) deployAntreaCommon(yamlFile string, extraOptions string, isAgent bool) error {
```

Comment:
why the parameter change? these 2 names mean completely different things, yet is seems that we are not updating any caller. Based on the code changes below, I am not sure what warrants a name change.